object/object.go

```diff
@@ -56,6 +56,7 @@ func Unmarshal[T any](raw []byte) *T {
 
 func Hash(o any) string {
 	h := fnv.New64a()
-	hash.DeepHashObject(h, o)
-	return strconv.FormatUint(uint64(h.Sum64()), 10)
+	h.Reset()
```

Comment:
This was needed to drop the Kubernetes dependency -- we shouldn't be relying on this repo as a dependency

Reply:
It mostly just broke GoLand for dependency resolution